cdap-ui/app/cdap/components/AdminMetadataPane/index.js

```diff
@@ -30,13 +31,23 @@ function AdminMetadataPane({ statObject }){
   let statsList = [];
 
   statObject.stats.forEach((stat) => {
-    statsList.push (
-      <StatContainer
-        label={T.translate(`features.Management.DetailPanel.labels.${stat.statName}`)}
-        number={stat.statNum}
-        key={shortid.generate()}
-      />
-    );
+
+    //Ignore Stats Associated with WritePointer, ReadPointer, and VisibilityUpperBound
+    if(!(stat.statName === 'WritePointer' || stat.statName === 'ReadPointer' || stat.statName === 'VisibilityUpperBound')) {
+
+      //Convert snapshot time to human readable number 
+      if(stat.statName === 'SnapshotTime'){
```

Comment:
hmmm, I don't think this stat makes sense, given that we're ignoring more important ones above. Its just the time that the stats were collected.